tmgmt_zanata.connector.inc

```diff
@@ -842,4 +875,88 @@ class TmgmtZanataConnector {
     );
   }
 
+  /**
+   * Function to format notification email body
+   */
+  private function formatEmailBody($email_contents, $type='PLAIN'){
+    if ($type === 'HTML') {
+      $msgBody = "<html><body>";
+      $msgBody .= "<h3>Hello,</h3>";
+      $msgBody .= "<br/><br/>";
+      $msgBody .= "A translation job has been submitted to Zanata. Details:";
```

Comment:
Can the title of the drupal site be included here, so we could say "... submitted to Zanata from <name of site>..."